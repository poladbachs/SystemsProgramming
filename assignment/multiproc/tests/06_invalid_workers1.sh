"$1" -w 0 ./tests/lorem_ipsum.txt >/dev/null
echo "Exit code is: $?"
