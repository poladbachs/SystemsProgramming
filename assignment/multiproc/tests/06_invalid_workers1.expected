Invalid number of workers provided. The number of workers must be > 0 and <= 20
Exit code is: 1
